File: performance-comparison-languages/find_primes.py

def is_prime(n):
    for i in range(2, n//2+1):
        if (not (n%i)):
            return 0
        
    return 1

num_primes = 0
upper_bound = 250001

for i in range(2, upper_bound):
    num_primes += is_prime(i)

print(str(num_primes))

File: performance-comparison-languages/find_primes.js

const isPrime = (n) => {
    for (let i = 2; i <= n/2; i++) {
        if (!(n % i)) {
            return 0;
        }
    }
    return 1;
};


var numPrimes = 0;
const upperBound = 250001;

for (let i = 2; i < upperBound; i++) {
    numPrimes += isPrime(i);
}

console.log(numPrimes);

